changelog/Microsoft.DirectoryServices.json

```
@@ -1,5 +1,89 @@
 {
-  "changelog": [	
+  "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "cf98cee8-873f-424f-9a0b-c683ea38a660",
+          "ApiChange": "Resource",
+          "ChangedApiName": "emailAuthenticationMethod",
+          "ChangeType": "Addition",
+          "Description": "Added the [emailAuthenticationMethod](https://docs.microsoft.com/graph/api/resources/emailAuthenticationMethod?view=graph-rest-1.0) resource and related methods. Use this resource and associated methods to manage a user's email authentication method for self-service password reset (SSPR).",
+          "Target": "emailAuthenticationMethod"
+        },
+        {
+          "Id": "341b6090-3a47-4710-8b57-88dae99b15e1",
+          "ApiChange": "Relationship",
+          "ChangedApiName": "emailAuthenticationMethod",
+          "ChangeType": "Addition",
+          "Description": "Added the **emailAuthenticationMethod** navigation property to the [authentication](https://docs.microsoft.com/graph/api/resources/authentication) resource.",
```

The nav property name is emailMethods, not emailAuthenticationMethod
```suggestion
          "Description": "Added the **emailMethods** navigation property to the [authentication](https://docs.microsoft.com/graph/api/resources/authentication) resource.",
```